pyproject.toml

```diff
@@ -79,12 +79,12 @@ pytest-cov = "^4.0.0"
 pytest-mock = "^3.5.1"
 flake8 = "^6.0.0"
 python-dotenv = "^0.21.0"
-black = {version = "^22.6.0", allow-prereleases = true}
+black = "^22.6.0"
```

Comment:
We want to update to `black v23.x.x` before beginning to run `black` on other modules. 
Requires updating to later `schematic_db` versions which will require updates to upsert functionality.

Reply:
`schematic_db` locked in #1145, 
looks like we need to make the switch from `uuid` to `id` in schematic to enable updating

@milen-sage @MiekoHash